### ISSUE TITLE
Mypy can't find types

### ISSUE THREAD

**Issue by sbdchd** [lk-geimfari/mimesis#977]
# Bug report

## What's wrong

```py
from mimesis import Person
person = Person('en')

assert isinstance(person.full_name(), str)
```

running Mypy I get:

```
❯ ./.venv/bin/mypy main.py
main.py:1:1: error: Skipping analyzing 'mimesis': found module but no type hints or library stubs  [import]
main.py:1:1: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 1 source file)
```

## How is that should be

Instead mypy should be able to find the type stubs.

I believe this is because the `package_data` is missing in the `setup.py`

https://www.python.org/dev/peps/pep-0561/#packaging-type-information

Maybe something like:

```diff
diff --git a/setup.py b/setup.py
index b3aa7fb..90055e3 100755
--- a/setup.py
+++ b/setup.py
@@ -136,6 +136,7 @@ setup(
     package_data={
         'mimesis': [
             'data/*/*',
+            'py.typed',
         ],
     },
     exclude_package_data={
```

I download the repo and built the wheel manually and when I install that 
mypy was able to find the types. Not sure exactly what the issue is but 
installing from PyPi seems a bit bugged.

Side note: building from `master` gave me version 4.0 instead of 4.1

The locally built wheel has the necessary `py.typed` file:

```
❯ ls .venv/lib/python3.7/site-packages/mimesis
__init__.py*   compat.py      enums.py       providers/     schema.py*
__pycache__/   data/          exceptions.py* py.typed       shortcuts.py*
builtins/      decorators.py* locales.py     random.py      typing.py
```

but the Pypi download doesn't:

```
❯ ls .venv/lib/python3.7/site-packages/mimesis
__init__.py    compat.py      enums.py       providers/     shortcuts.py
__pycache__/   data/          exceptions.py  random.py      typing.py
builtins/      decorators.py  locales.py     schema.py
```

## System information

python = "^3.7"
mimesis = "^4.1"

**Comment by MrGreenTea** [comment]
I assume this is because [py.typed](https://github.com/lk-geimfari/mimesis/blob/master/mimesis/py.typed) is not included in the release.

**Comment by MrGreenTea** [comment]
@lk-geimfari Will you release this fix in the near future?

**Comment by lk-geimfari** [comment]
Sure. I'll release it today or tomorrow that's for sure.

**Comment by lk-geimfari** [comment]
@MrGreenTea I've released 4.1.3.
